### DISCUSSION TITLE
Simulation Module (-S) segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by a3490489** [neperfepx/neper#730] [Q&A] [answered]
Dear all

Thank you for reading this Q&A.
I'm trying to install Neper 4.6.1-7 in WSL2 and VMware (Ubuntu 22.04.2 LTS).
Everything ran well except the Simulation module (-S) when I wanted to post-process the results file of FEPX.
When I run the example 1_uniaxial in FEPX, "segmentation fault (core dumped)" happened during post-process (_neper -S ._).
```
tate@tate-virtual-machine:~/Softwares/FEPX/examples/1_uniaxial$ neper -S .

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.6.1-7
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 2 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/tate/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -S loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] .
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Converting FEPX result directory into simulation directory...
Info   :   - Parsing FEPX results...
Segmentation fault (core dumped)

```
I tried using a single CPU or multi-CPU, but the results were the same.
Does anyone know where the problem is and how to fix it?
Thank you very much, and I'm looking forward to your kind help.

Sincerely,
Ta-Te

**Comment by rquey** [comment, marked as answer]
I confirm that it works for me, on standard Ubuntu 22.04.  So, it may have to do with "WSL2 and VMware".
